app_feup/lib/view/Theme.dart

```diff
@@ -7,6 +7,7 @@ const Color greyTextColor = Color.fromARGB(255, 0x46, 0x46, 0x46);
 const Color whiteTextColor = Color.fromARGB(255, 255, 255, 255);
 const Color bodyTitle = Color.fromARGB(255, 131, 131, 131);
 const Color greyBorder = Color.fromARGB(64, 0x46, 0x46, 0x46);
+const Color lightGreyTextColor = Color.fromARGB(255, 150, 150, 150);
```

Comment:
It does seem like this sort of approach to show update time will be replicated in other features, wouldn't it be best to save as a new option under the theme data itself?

Reply:
Saved it as indicatorColor.